client/android/WorldScope/app/src/main/java/com/litmus/worldscope/FacebookLoginActivity.java

```diff
@@ -1,27 +1,81 @@
 package com.litmus.worldscope;
 
+import android.content.Context;
+import android.content.Intent;
 import android.support.v4.app.FragmentActivity;
 import android.os.Bundle;
 import android.util.Log;
+import android.widget.Toast;
 
 import com.facebook.AccessToken;
 
 import layout.FacebookLoginFragment;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 
 public class FacebookLoginActivity extends FragmentActivity implements FacebookLoginFragment.OnFragmentInteractionListener {
```

Comment:
If we are using the v7 support library, I think using `extends AppCompatActivity` would be a better choice. AppCompatActivity is the direct child class of FragmentActivity of support v4. [stackoverflow link](http://stackoverflow.com/questions/31297246/activity-appcompatactivity-fragmentactivity-and-actionbaractivity-when-to-us)